### PR TITLE
Add 'URL Bar Resizing' article.

### PR DESCRIPTION
**Pull request by jpmedley**

### CODE REVIEW

File: src/content/en/updates/2016/12/url-bar-resizing.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+to the URL bar being shown or hidden. Instead, `vh` units will be sized to the
+viewport height as if the URL bar is always hidden. That is, `vh` units will be
+sized to the "largest possible viewport". This means `100vh` will be larger
+than the visible height when the URL bar is shown.
+
+The Initial Containing Block (ICB) is the root containing block used when
+sizing elements relative to their parents. For example, giving the `<html>`
+element a style of `width: 100%; height: 100%` will make it the same size as
+the ICB. With this change, the ICB will not resize when the URL bar is hidden.
+In stead, it will remain the same height, as if the URL bar were always showing
```
Comment (bokand) [inline src/content/en/updates/2016/12/url-bar-resizing.md]: "In stead" -> Instead

**Comment by bokand** [comment]
lgtm % typo above

File: src/content/en/updates/2016/12/url-bar-resizing.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+description: Hiding the URL bar no longer resizes mobile pages.
+
+{# wf_updated_on: 2016-12-16 #}
+{# wf_published_on: 2016-12-16 #}
+{# wf_tags: chrome56,android,ux #}
+{# wf_featured_image: /web/updates/images/generic/quilt.png #}
+{# wf_featured_snippet: Hiding the URL bar no longer resizes mobile pages. #}
+
+# URL Bar Resizing {: .page-title }
+
```
Comment (petele) [inline src/content/en/updates/2016/12/url-bar-resizing.md]: Should include the author widget here

**Comment by petele** [comment]
Replaced with #3982

**Pull request closed without merging**